### PR TITLE
Allow custom images in empty state component

### PR DESCRIPTION
**Pull request by Joonpark13**
## Hey, I just made a Pull Request!

See title.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by emmaindal — COMMENTED** [review]
Can you add this option as a storybook example too https://backstage.io/storybook/?path=/story/feedback-emptystate--content

here is the storybook file https://github.com/backstage/backstage/blob/master/packages/core-components/src/components/EmptyState/EmptyState.stories.tsx

File: packages/core-components/src/components/EmptyState/EmptyState.tsx [hunk 1]
```diff
@@ -42,9 +42,14 @@ const useStyles = makeStyles(
 type Props = {
   title: string;
   description?: string | JSX.Element;
-  missing: 'field' | 'info' | 'content' | 'data';
```
Comment (freben) [inline packages/core-components/src/components/EmptyState/EmptyState.tsx:45]: hypothetical alternative:

```ts
missing: 'field' | 'info' | 'content' | 'data' | { customImage: JSX.Element };
```

🤷

Comment (Joonpark13) [reply packages/core-components/src/components/EmptyState/EmptyState.tsx:45]: Done!

**Review by freben — APPROVED** [review]
Aight!

**Pull request merged**